dev/com.ibm.ws.session.cache_fat/publish/servers/sessionCacheServer/server.xml

```diff
@@ -29,23 +29,25 @@
 	    <jdbcDriver libraryRef="DerbyLib"/>
 	    <properties.derby.embedded createDatabase="create" databaseName="memory:testdb"/>
 	</dataSource>
+	
+	<!-- Perms needed because the application uses Hazelcast directly sometimes -->
```

Comment:
Should these be moved to the application so that they don't hide errors with missing doPriv in Liberty product code?